CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - [#1496](https://github.com/JabRef/jabref/issues/1496) Keep track of which entry a downloaded file belongs to
 - Made it possible to download multiple entries in one action
 - [#1813](https://github.com/JabRef/jabref/issues/1813) Import/Export preferences dialog default directory set to working directory
+- [#1897](https://github.com/JabRef/jabref/issues/1897) Added last four nonpunctuation characters should be numerals in Check Integrity year
```

Comment:
I would reword it a bit to make it a a bit more clear: 
>  Implemented integrity check for year field: Last four nonpunctuation characters should be numerals 

(Put the word "year" in backticks (the french accent grave), just look at other examples in the changelog)